staging/src/k8s.io/client-go/tools/events/event_broadcaster.go

```diff
@@ -102,16 +102,20 @@ func newBroadcaster(sink EventSink, sleepDuration time.Duration, eventCache map[
 	}
 }
 
+func (eventBroadcaster *eventBroadcasterImpl) Shutdown() {
+	eventBroadcaster.Broadcaster.Shutdown()
+}
+
 // refreshExistingEventSeries refresh events TTL
-func (e *eventBroadcasterImpl) refreshExistingEventSeries() {
+func (eventBroadcaster *eventBroadcasterImpl) refreshExistingEventSeries() {
```

Comment:
If you want to change names, please do that in a separate commit.
[That said, I actually think that "e" was better than such a long name.

Reply:
@wojtek-t 
I have taken out the name change.
Please check again.